### PR TITLE
feat: added color mode

### PR DESCRIPTION
**Pull request by Parkuel**

### CODE REVIEW

File: src/theme/theme.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { extendTheme } from '@chakra-ui/react';
+
+const config = {
+  initialColorMode: 'light',
```
Comment (itsnotpolza) [inline src/theme/theme.js:4]: use system color mode as default

Comment (itsnotpolza) [inline src/theme/theme.js:4]:
```suggestion
  initialColorMode: 'system',
```

File: src/theme/colorModeButton.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Button } from '@chakra-ui/react';
+import { useColorMode } from '@chakra-ui/react';
+
+export const ColorModeScript = () => {
```
Comment (itsnotpolza) [inline src/theme/colorModeButton.jsx:4]: Rename to ColorModeToggle

File: src/main.jsx [hunk 3]
```diff
@@ -3,10 +3,13 @@ import ReactDOM from 'react-dom/client';
 import { App } from './pages';
 import './index.css';
 import { ChakraProvider } from '@chakra-ui/react';
+import { ColorModeScript } from './theme/colorModeButton';
```
Comment (itsnotpolza) [inline src/main.jsx:6]: export component from theme folder (using index.js) and name of the file should be the same as the component that's being exported from it

Comment (Parkuel) [reply src/main.jsx]: index.js has been created and the imports have been shoretened

File: src/theme/colorModeButton.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { Button } from '@chakra-ui/react';
```
Comment (itsnotpolza) [inline src/theme/colorModeButton.jsx:1]: rename file to the same name as the component

**Review by kalashnikau — APPROVED** [review]

**Review by lazzzaroni — APPROVED** [review]
Fingers crossed! =)

**Pull request merged**